helpers.js

```diff
@@ -36,3 +36,16 @@ export const formatDateDiff = (recent, old) => {
     day: "numeric",
   }).format(old.getTime()); // ex: Nov 15
 };
+
+export function formatLargeNumber(n) {
+  let s = "";
+  if (n < 1000) {
+    s = n;
+  } else if (n >= 1000 && n < 1000000) {
+    s = (Math.round(n)/1000).toFixed(1) + "k"
```

Comment:
This does not work when `n = 999,999`. The expected result is `999k` but actual result is `1000.0k`
The same problem with `n = 499,950` => `499.9k`, but `n = 499,951` => `500.0k`.